src/server/snap.rs

```diff
@@ -154,8 +151,10 @@ fn send_snap(
     };
 
     let cb = ChannelBuilder::new(env)
```

Comment:
Why gRPC related things changed?

Reply:
These configurations are used to optimize the transport of snapshot just like send/recv limit.